### DISCUSSION TITLE
up-nav: Set up-current for links that start with location

### DISCUSSION THREAD

**Discussion by robbevp** [unpoly/unpoly#205] [Q&A] [answered]
Hi there 👋 
I'm trying out unpoly for the first time.

One of the things I bumped into is that my navigation is not styled based on the exact match, but should if the href matches the start of the location.
If my nav contains two links:
`/foo`
`/bar`

Foo should be styled if the user's location is `/foo`, `/foo/child`, `/foo/child/grandchild`, ...

Is there a way to do this with up-nav? OR would you be open for a PR to add this option?

**Comment by triskweline** [comment, marked as answer]
Hi @robbevp ! Thanks for trying out Unpoly and offering to contribute.

To match additional URLs you can put an [URL pattern](https://unpoly.com/url-patterns) into the [`[up-alias]`](https://unpoly.com/a-up-alias) attribute. So in your example:

```html
<nav>
  <a href="/foo" up-alias="/foo/*">...</a>
  <a href="/bar" up-alias="/bar/*">...</a>
</nav>
```

**Reply by robbevp** [reply]
Thanks @triskweline
